frontend/public/components/utils/file-input.tsx

```diff
@@ -42,7 +42,7 @@ class FileInputWithTranslation extends React.Component<FileInputProps, FileInput
         ? (reader.result as string).split(',')[1]
         : (reader.result as string);
       // OnLoad, if inputFileIsBinary we have read as a binary string, skip next block
-      if (isBinary(null, input) && !fileIsBinary) {
+      if (isBinary(null, reader.result as Buffer) && !fileIsBinary) {
```

Comment:
`reader.result` is not a `Buffer`. It is either a `string` or an `ArrayBuffer`, which is not the same as `Buffer`;
```suggestion
      if (isBinary(null, Buffer.from(reader.result)) && !fileIsBinary) {
```